src/test/java/api/support/fakes/FakeCalendarOkapi.java

```diff
@@ -5,116 +5,101 @@
-            routingContext.response()
-              .putHeader("content-type", "application/json")
-              .setStatusCode(404)
-              .end();
-            break;
-
-          default:
-            MultiMap queries = routingContext.queryParams();
-            routingContext.response()
-              .setStatusCode(200)
-              .putHeader("content-type", "application/json")
-              .end(findFakeCalendarById(servicePointId, queries));
+  public static void registerCalendarSurroundingDates(Router router) {
+    router
+      .get("/calendar/dates/:id/surrounding-openings")
+      .handler(
+        routingContext -> {
+          String servicePointId = routingContext.pathParam("id");
+          switch (servicePointId) {
+            case CASE_CALENDAR_IS_UNAVAILABLE_SERVICE_POINT_ID:
+              routingContext
+                .response()
+                .putHeader("content-type", "application/json")
+                .setStatusCode(404)
+                .end();
+              break;
+            case CASE_CLOSED_LIBRARY_SERVICE_POINT_ID:
+              routingContext
+                .response()
+                .putHeader("content-type", "application/json")
+                .setStatusCode(404)
+                .end();
+              break;
+            case CASE_CALENDAR_IS_EMPTY_SERVICE_POINT_ID:
+              routingContext
+                .response()
+                .putHeader("content-type", "application/json")
+                .setStatusCode(200)
+                .end();
+              break;
+            case CASE_CLOSED_LIBRARY_IN_THU_SERVICE_POINT_ID:
+              routingContext
+                .response()
+                .putHeader("content-type", "application/json")
+                .setStatusCode(404)
+                .end();
+              break;
+            default:
+              MultiMap queries = routingContext.queryParams();
+              routingContext
+                .response()
+                .setStatusCode(200)
+                .putHeader("content-type", "application/json")
+                .end(findFakeCalendarById(servicePointId, queries));
+          }
         }
-      });
-  }
-
-  private static String findFakeLibraryHoursById(String servicePointId) {
-    log.debug(String.format("GET: /calendar/periods/%s/period", servicePointId));
-    return getLibraryHoursById(servicePointId).toString();
+      );
   }
 
-  private static String findFakeCalendarById(String servicePointId, MultiMap queries) {
-    log.debug(String.format("GET: /calendar/periods/%s/calculateopening, queries=%s",
-      servicePointId, queries));
-    return getCalendarById(servicePointId, queries).toString();
+  private static String findFakeCalendarById(
+    String servicePointId,
+    MultiMap queries
+  ) {
+    log.info(
+      String.format(
+        "GET: /calendar/dates/%s/surrounding-openings, queries=%s",
+        servicePointId,
+        queries
+      )
+    );
```

Comment:
You can use logger's built-in formatting instead of `String#format`:
```suggestion
    log.info("GET: /calendar/dates/{}/surrounding-openings, queries={}", servicePointId, queries));
```